packages/actor-init-sparql/lib/ActorInitSparql-browser.ts

```diff
@@ -84,6 +87,38 @@ export class ActorInitSparql extends ActorInit implements IActorInitSparqlArgs {
     super(args);
   }
 
+  public static enhanceQueryResults(results: IQueryResult): IQueryResult {
```

Comment:
Add tsdoc comment above, such as _Add convenience methods to query results_, with a description of the param (and return type).